Dockerfile-fpm

```diff
@@ -9,6 +9,7 @@ ENV PHP_FPM_PM_MIN_SPARE_SERVERS=1
 ENV PHP_FPM_PM_MAX_SPARE_SERVERS=3
 ENV PHP_FPM_PM_PROCESS_IDLE_TIMEOUT=10
 ENV PHP_FPM_PM_MAX_REQUESTS=0
+ENV PHP_FPM_ACCESS_FORMAT %R - %u %t \\\"%m %r\\\" %s
```

Comment:
This works without a `=`?
Can we add the equal sign just for the sake of consistency?

Reply:
Yeah it works without the `=` and the `"` around it. Just added them again and tested it, it works but hadolint still chokes on it :joy: . So this line with without `=` and `"` is a compromise between Docker, hadolint, and me. Can do the same for the other lines, for consistency sake